### DISCUSSION TITLE
The readme says `sourcekit` can be set to `nil` to skip lspconfig set up, but isn't that the equivalent of not setting the value at all?

### DISCUSSION THREAD

**Discussion by heygarrett** [kkharji/xbase#232] [Q&A] [answered]
From the Neovim defaults section of the readme:

```lua
  --- Options to be passed to lspconfig.nvim's sourcekit setup function.
  --- Usually empty map is sufficient, However, it is strongly recommended to use on_attach key to setup custom mapppings
  sourcekit = {}, --- Set it to nil to skip lspconfig's sourcekit setup
```

In Lua a table's key being set to `nil` effectively removes that key from the table. In this case wouldn't that mean the `sourcekit` key takes on the default value of `{}`? In other words, there's not currently a way to skip lspconfig's sourcekit setup.

**Comment by kkharji** [comment, marked as answer]
https://github.com/kkharji/xbase/blob/d3e7f51f17a8dbc5049ebf74c63598ca5162f01f/lua/xbase/init.lua#L38C11-L38C11

Oh yeah 100%. The default should be nil. This would be simple change. 

Fixed in https://github.com/kkharji/xbase/pull/233

Thanks for reporting @heygarrett
